src/components/PopupQuestion.js

```diff
@@ -0,0 +1,23 @@
+import Popup from "./Popup.js";
```

Comment:
Please, delete the close icon handler from line 33 in `Popup.js`, because you handle it in line 40 with the overlay

Reply:
Please, delete the close icon handler from line 35 in `Popup.js`, because you handle it in line 42 with the overlay

Reply:
Hi! Just did thank you